### ISSUE TITLE
Learning Rate Scheduler

### ISSUE THREAD

**Issue by nishithreen24** [nishithreen24/Deep-Learning#1]
The ANN requires a learning rate scheduler: Step/Exponential Decay/Cosine Annealing for training on non-convex training sets (CIFAR-10/Fashion MNIST) where more epochs will be required.
